### PR TITLE
The Weekendest: Add option to only show certain lines

### PR DESCRIPTION
**Pull request by josephschmitt**
Allows you to set a comma-separated list of lines (e.g. `"A,C"`) to filter to show instead of all the lines of the station.
image

## Before
image

## After
image

### CODE REVIEW

**Comment by josephschmitt** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by matslina** [comment]
@blahblahblah- Ok to merge?

**Comment by blahblahblah-** [comment]
Small feedback, but LGTM otherwise!

File: apps/goodservice/goodservice.star [hunk 1]
```diff
@@ -139,6 +138,14 @@ def main(config):
                         blocks.append(render.Box(width = 64, height = 1, color = "#333"))
 
                 selected_route = routes_req.json()["routes"][r["route_id"]]
+
+                include_lines_str = config.str("include_lines", "").lower()
+                include_lines = include_lines_str.split(",")
```
[the remark this answers is not in the document]
Comment (josephschmitt) [reply apps/goodservice/goodservice.star:143]: Sure can, I'll move them to the top of the fn

**Comment by josephschmitt** [comment]
> Small feedback, but LGTM otherwise!

@blahblahblah- @matslina Updated

**Comment by blahblahblah-** [comment]
LGTM!

**Pull request merged**